packages/vue/src/components/IonRouterOutlet.ts

```diff
@@ -198,7 +196,7 @@ export const IonRouterOutlet = /*@__PURE__*/ defineComponent({
     const transition = (
       enteringEl: HTMLElement,
       leavingEl: HTMLElement,
-      direction: any, // TODO types
+      direction: any,
```

Comment:
If we are removing the todo statement for types, should we add the type here or create a new ticket to track?

Reply:
I've got FW-2690 in this sprint, which is to revisit `any` types across the codebase. I figure I can handle it then.